src/main/java/edu/cornell/kfs/concur/service/impl/ConcurEventNotificationConversionServiceImpl.java

```diff
@@ -15,10 +15,13 @@
     public ConcurEventNotification convertConcurEventNotification(ConcurEventNotificationDTO concurEventNotificationDTO) throws ParseException {
         ConcurEventNotification concurEventNotification = new ConcurEventNotification();
         concurEventNotification.setContext(concurEventNotificationDTO.getContext());
-        concurEventNotification.setEventDateTime(dateTimeService.convertToSqlDate(concurEventNotificationDTO.getEventDateTime()));
+        if(concurEventNotificationDTO.getEventDateTime() != null){
```

Comment:
This is a field in the incoming payload xml so checking for null is enough, this is not a check on a proxied object that may or may not be materialized yet so ObjectUtils would not work here.